staging/src/k8s.io/cloud-provider/controllers/service/controller.go

```diff
@@ -772,24 +748,29 @@ func (c *Controller) nodeSyncInternal(ctx context.Context, workers int) {
 		numServices-len(c.servicesToUpdate), numServices)
 }
 
-// nodeSyncService syncs the nodes for one load balancer type service
-func (c *Controller) nodeSyncService(svc *v1.Service) bool {
+// nodeSyncService syncs the nodes for one load balancer type service. The return value
+// indicates if we should retry. Hence, this functions returns false if we've updated
+// load balancers and finished doing it successfully, or didn't try to at all because
+// there's no need. This function returns true if we tried to update load balancers and
+// failed, indicating to the caller that we should try again.
+func (c *Controller) nodeSyncService(svc *v1.Service, oldNodes, newNodes []*v1.Node) bool {
+	retSuccess := false
+	retNeedRetry := true
```

Comment:
it looks like these are intended to be named constants?
```suggestion
	const retSuccess = false
	const retNeedRetry = true
```

Reply:
Though I would say the more usual approach would be to let the method documentation make it clear what true and false mean.

Reply:
Would you say the current state of that doc bloc does a good job at making it clear what true and false mean? In that case, I could just remove this in a follow up PR.

Reply:
It's a weird signature (false usually means fail) so I like having the const values.